spec/tests/requires.fmf

```diff
@@ -0,0 +1,29 @@
+summary: Packages required for the test execution
+
+story:
+    As a tester I want to specify packages which are required by
+    the test and need to be installed on the system so that the
+    test can be successfully executed.
+
+description: |
+    In order to execute the test, additional packages may need to
+    be installed on the system. For example `gcc` and `make` are
+    needed to compile tests written in C on the target machine. If
+    the package cannot be installed test execution should result
+    in an ``error``.
+
+    Packages being tested (system under test) should not be listed
```

Comment:
while I agree with this, we see now a problem with this approach with STI tests and automatic execution of these tests as integration tests for other components. If we do not add those packages here, we will need another field, as otherwise we will get into the same trouble as with STI and reversedependency

CC @bgconcalv

Reply:
I see. But the question is the second part of the description: What if...

> there are multiple packages covered by a single test (regular package, module, software collection) or the component has different names in different product versions?

Then tests cannot be shared. I would suggest that for integration testing test would be executed including L2 Metadata. Would that help here?

Reply:
Thanks for the feedback. So here's what I propose: What about giving just a recommendation for shared tests. It could be something like this:

> For tests shared across multiple product versions where required packages have different names it is recommended to use the `prepare` step configuration to specify required packages for each product version individually.

Does it now make more sense?

Reply:
Description updated to use the wording above.

Reply:
it looks better :) thanks.